flutter/shell/platform/tizen/BUILD.gn

```diff
@@ -174,15 +179,26 @@ template("embedder") {
     defines += invoker.defines
     defines += [ "FLUTTER_ENGINE_NO_PROTOTYPES" ]
 
-    if (api_version == "6.5" && target_name != "flutter_tizen_wearable") {
+    if (api_version == "6.5") {
```

Comment:
`api_version` is expected to be either 5.5 or 6.5. No other values are allowed at the moment.

Reply:
I think I misunderstood it. it's modified.